pkg/proxy/nginx/nginx.go

```diff
@@ -111,8 +112,12 @@ func (f *nginx) Run(stopCh <-chan struct{}) {
 
 	log.Infof("Starting nginx proxy, workers %v, image %v, default-http-backend %v, sidecar %v", workers, f.image, f.defaultHTTPbackend, f.sidecar)
 
-	if err := f.ensureDefaultHTTPBackend(); err != nil {
-		panic(fmt.Sprintf("Ensure default http backend service error, %v", err))
+	if runtime.GOARCH == "amd64" {
+		if err := f.ensureDefaultHTTPBackend(); err != nil {
+			panic(fmt.Sprintf("Ensure default http backend service error, %v", err))
+		}
+	} else {
+		log.Infof("Arch is %s, default-http-backend is deprecated", runtime.GOARCH)
```

Comment:
这里优化下：当 f.defaultHTTPbackend 为默认值，不应该报“deprecated”，应该 info
"Arch is %s, use internal default backend"；当  f.defaultHTTPbackend被传值了，报warning deprecated 相关内容。

Reply:
这里无法传入 f.defaultHTTPbackend 的默认值（变量被定义在另外一个 package），而上面又有 print  f.defaultHTTPbackend，所以我换了个说法：use internal default-http-backend instead of f.defaultHTTPbackend，不管是默认值还是传递值都可以体现